src/index.html

```diff
@@ -1,15 +1,195 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>HTML Form</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>HTML Form</h1>
-  <script type="text/javascript" src="./main.js"></script>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>HTML Form</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="post"
+    autocomplete="off"
+    >
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+        <label class="form__item" for="surname">
+          Surname:
+          <input
+            type="text"
+            name="surname"
+            id="surname"
+            required
+          >
+        </label>
+
+        <label class="form__item" for="name">
+          Name:
+          <input
+            type="text"
+            name="name"
+            id="name"
+            required
+          >
+        </label>
+
+        <label class="form__item" for="old">
+          How old are You?
+          <input
+            type="number"
+            min="10"
+            max="100"
+            name="old"
+            id="old"
+            value="12"
+          >
+        </label>
+
+        <label class="form__item" for="birth_date">
+          Full date of birth:
+          <input
+            type="date"
+            name="birth_date"
+            id="birth_date"
+          >
+        </label>
+
+        <label class="form__item" for="agreement"
+        required>
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="agreement"
+            id="agreement"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>Registration</legend>
+        <label class="form__item" for="email">
+          E-mail:
+          <input
+            type="email"
+            name="email"
+            placeholder="email@example.com"
+            id="email"
+            required
+          >
+        </label>
+
+        <label class="form__item" for="password"
+        minlength="8" maxlength="20">
```

Comment:
minlength and maxlength used only with inputs
```suggestion
        <label class="form__item" for="password">
```